src/components/util/Cards.js

```diff
@@ -28,16 +28,18 @@ class Cards extends Component {
             }
             return (
                 <div className='card' key={index}>
-                    <h4
-                        key={index + 100}
```

Comment:
We should indicate somewhere that this code will only allow maximum of 100 cards to render, or is there a better way to implement an unique key?

Reply:
Good call.